yetanotherco/aligned_layer
# 1201 make risc0 public input optional

closes #1201 

## To Test:
- Run local testnet
- Submit risc0 proofs to the local batcher via `make batcher_send_risc0_burst`. Observe they are processed.
- Submit risc0 proof with empty public inputs to the local batcher via `make batcher_send_risc0_task_no_pub_input`. Observe they are processed.
